### PR TITLE
build: Update build process for publish

### PR DESCRIPTION
**Pull request by zombieJ**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Branch merge
- [x] Other: Script & Publish update

### 👻 What's the background?

ref: https://github.com/ant-design/antd-tools/pull/96

### 💡 Solution

1. Use `.antd-tools.config.js` to config `dist` follow up
2. Add diff-check lint for publish package check

### CODE REVIEW

File: .antd-tools.config.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+const fs = require('fs');
+const path = require('path');
+const packageInfo = require('./package.json');
+
+// We need compile additional content for antd user
+function finalizeDist() {
+	if (fs.existsSync(path.join(__dirname, './lib'))) {
+		// Build package.json version to lib/version/index.js
```
Comment (afc163) [inline .antd-tools.config.js]: 缩进有点问题。

**Review by ztplz — APPROVED** [review]

File: tests/dekko/dist.test.js [hunk 2]
```diff
@@ -1,12 +1,22 @@
...
+  .hasFile('components.less');
+
 $('dist')
   .isDirectory()
   .hasFile('antd.css')
   .hasFile('antd.min.css')
   .hasFile('antd.js')
-  .hasFile('antd.min.js');
+  .hasFile('antd.min.js')
+  .hasFile('antd.less');
```
Comment (afc163) [inline tests/dekko/dist.test.js]: 其他文件也查一下。

**Review by afc163 — APPROVED** [review]

**Comment by afc163** [comment]
合并后试发一个 beta

**Comment by zombieJ** [comment]
过了，看起来没问题：https://unpkg.com/antd@3.16.2-alpha.0/

**Pull request merged**